tests/ui/lint/non-snake-case/lint-non-snake-case-crate.rs

```diff
@@ -22,6 +22,10 @@
 //@[rlib_] compile-flags: --crate-type=rlib
 //@[staticlib_] compile-flags: --crate-type=staticlib
 
+// The compiler may emit a warning if the target does not support unwinding, so
+// ignore output in addition to the expected error.
+//@[proc_macro_] compare-output-lines-by-subset
```

Comment:
You can also just do the simple thing and `//@ needs-unwind` to skip targets that don't support unwinding

Reply:
I had considered that approach but figured compare-output-lines-by-subset was a decent comprise to have some coverage on both kinds of targets. I'm happy to go with either approach. WDYT?

Reply:
I personally prefer just skipping it instead of using `compare-output-lines-by-subset`, I'll let the assigned reviewer decide.